packages/angular-cli/blueprints/ng2/files/angular-cli.json

```diff
@@ -8,12 +8,15 @@
       "root": "<%= sourceDir %>",
       "outDir": "dist",
       "assets": "assets",
-      "index": "index.html",
-      "main": "main.ts",
+      "index": "index.html",<% if(!universal) { %>
+      "main": "main.ts",<% } %><% if(universal) { %>
+      "main": "client.ts",
```

Comment:
I did not want to change so much in the CLI code. We can talk about a strategy to rename files. Because I need the main.ts file as client.ts in the universal context. This was the easiest solution for me to not implement "renaming files".